mockkubeapiserver/storage/memorystorage/crd.go

```diff
@@ -0,0 +1,101 @@
+package memorystorage
+
+import (
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/kubebuilder-declarative-pattern/mockkubeapiserver/storage"
+)
+
+// UpdateCRD is called whenever a CRD is updated.
+// We register the types from the CRD.
+func (s *MemoryStorage) UpdateCRD(ev *storage.WatchEvent) error {
+	// TODO: Deleted / changed CRDs
```

Comment:
What is the expected behavior of "delete CRD". Is this for the case when CRD belongs to a versioned manifest?

Reply:
If a CRD is deleted, we should unregister it from the apiserver, so it isn't discoverable, you can't query it etc.  That's ... not implemented yet!